src/analysis/fourier/riemann_lebesgue_lemma.lean

```diff
@@ -60,10 +60,14 @@ variables [normed_add_comm_group V] [measurable_space V] [borel_space V]
 lemma fourier_integrand_integrable (w : V) :
   integrable f ↔ integrable (λ v : V, e [-⟪v, w⟫] • f v) :=
 begin
-  have hL : continuous (λ p : V × V, bilin_form_of_real_inner.to_lin p.1 p.2) := continuous_inner,
+  have hL : continuous (λ p : V × V, innerₛₗ ℝ p.1 p.2) := continuous_inner,
+  -- TODO: remove the continuity assumption from `vector_fourier.fourier_integral_convergent_iff`
```

Comment:
I don't think the lemma is still true without this assumption, as you can get measurability issues, no?

Reply:
I haven't checked the details, but I agree that one probably has to assume measurability.